### PR TITLE
added remove two's function

### PR DESCRIPTION
**Pull request by Soat6041**
We created a function that takes a number divides it by two until it cannot be divided by 2 any more. After, it will return the number that is no longer divisible by two.

### CODE REVIEW

**Comment by shro5435** [comment]
Interesting way to remove odds. Nice code!

File: math/FunMath.cpp [hunk 1]
```diff
@@ -1,5 +1,17 @@
 #include "FunMath.h"
 
+using namespace std;
+
+
+
```
Comment (muzny) [inline math/FunMath.cpp]: You need a function comment describing what this function does, what parameters it uses and what its return value is.

File: math/FunMath.cpp [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+int removeTwos(int myInt){
```
Comment (muzny) [inline math/FunMath.cpp]: Follow our naming conventions in our style guide: RemoveTwos and my_int

**Comment by chakryaros** [comment]
I have merged Remove2Function to master successfully and delete Remove2Function.

**Pull request merged**